rasa/utils/tensorflow/rasa_layers.py

```diff
@@ -805,6 +806,7 @@ def _get_transformer_dimensions(
         transformer_units = config[TRANSFORMER_SIZE]
         if isinstance(transformer_units, dict):
             transformer_units = transformer_units[attribute]
+        transformer_units = transformer_units or DEFAULT_TRANSFORMER_SIZE
```

Comment:
Nitpicking: would it be more readable if this would be rewritten using the `if/else` ternary operator?

Reply:
you mean changing to this?
`transformer_units = transformer_units if transformer_units else DEFAULT_TRANSFORMER_SIZE`

Reply:
The existing [check](https://github.com/RasaHQ/rasa/blob/main/rasa/nlu/selectors/response_selector.py#L394) in `ResponseSelector._warn_and_correct_transfomer_size` tests for both `transformer_size is None` and `transformer_size < 1`, which allows to correct configs like this [test case](https://github.com/RasaHQ/rasa/blob/main/tests/nlu/selectors/test_selectors.py#L556), where `transformer_size = -1` is expected to be corrected to the default size. To be consistent the same check should be applied in the fix here.